### ISSUE TITLE
migrate create -g can't be a relative path

### ISSUE THREAD

**Issue by vlindhol** [tj/node-migrate#175]
I am trying to create my own template generator as per the documentation.

```
migrate create -g ./src/migrations/generator.js foo
# leads to "module not found" error
```

The problem is that the library takes the argument to `-g` and passes it directly into `require`, and `require` doesn't respect `chdir` etc, meaning that a relative path is relative to the source file we are currently in. This is probably never something we want.

This is a trivial fix, will make a pull request.

**Comment by vlindhol** [comment]
Went to fix this and turns out it has been fixed in the code (the `require` is wrapped in a try/catch), but the latest release (1.6.2) doesn't incorporate it! Can a new release be made?

**Comment by wesleytodd** [comment]
Fixed in `1.7.0`
